src/js/views/Dashboard/ActivityFeed/Feed.jsx

```diff
@@ -1,47 +1,66 @@
 import PropTypes from 'prop-types'
 import React, { useContext, useEffect, useState } from 'react'
 import { useTranslation } from 'react-i18next'
+import InfiniteScroll from 'react-infinite-scroll-component'
 
-import { Alert, ContentArea, ErrorBoundary, Panel } from '../../../components'
+import {
+  Alert,
+  ContentArea,
+  ErrorBoundary,
+  Loading,
+  Panel
+} from '../../../components'
 import { Context } from '../../../state'
-import { httpGet } from '../../../utils'
+import { httpGet, parseLinkHeader } from '../../../utils'
 
-import { Entry } from './Entry'
+import { ActivityEntry } from './ActivityEntry'
+import { OpsLogEntry } from './OpsLogEntry'
 
 function Feed({ onReady }) {
   const [globalState] = useContext(Context)
   const [state, setState] = useState({
     data: [],
     fetched: false,
-    errorMessage: null
+    errorMessage: null,
+    nextLink: new URL('/activity-feed?omit_user=SonarQube', globalState.baseURL)
   })
   const { t } = useTranslation()
 
   useEffect(() => {
-    if (state.fetched === false) {
-      const url = new URL('/activity-feed', globalState.baseURL)
-      httpGet(
-        globalState.fetch,
-        url,
-        (result) => {
-          // defaulting 'type' to 'ProjectFeedType' for backwards compatibility
-          setState({
-            data: result.filter(
-              (f) => (f.type || 'ProjectFeedEntry') === 'ProjectFeedEntry'
-            ),
-            fetched: true,
-            errorMessage: null
-          })
-        },
-        (error) => {
-          setState({ data: [], fetched: true, errorMessage: error })
-        }
-      )
+    if (!state.fetched) {
+      fetchNext()
     } else {
       onReady()
     }
   }, [state.fetched])
 
+  function fetchNext() {
+    httpGet(
+      globalState.fetch,
+      state.nextLink,
+      ({ data, headers }) => {
+        const links = parseLinkHeader(headers.get('Link'))
+        const next = Object.hasOwn(links, 'next') ? links.next[0] : null
+        setState((prevState) => ({
+          data: prevState.data.concat(data),
+          fetched: true,
+          errorMessage: null,
+          nextLink: next
+            ? new URL(`${next}&omit_user=SonarQube`, globalState.baseURL)
```

Comment:
*sigh* .... our configured the SonarQube user has a username of `sonarqube` and a display name of `SonarQube`.  The query param is the user name.

Reply:
Ah, oops. Good catch.